src/cljs/athens/effects.cljs

```diff
@@ -369,21 +374,64 @@
                    100)))
 
 
-(reg-fx
+(rf/reg-fx
   :stylefy/tag
   (fn [[tag properties]]
     (stylefy/tag tag properties)))
 
 
-(reg-fx
+(rf/reg-fx
   :alert/js!
   (fn [message]
     (js/alert message)))
 
 
-(reg-fx
+(rf/reg-fx
   :right-sidebar/scroll-top
   (fn []
     (let [right-sidebar (js/document.querySelector ".right-sidebar-content")]
       (when right-sidebar
         (set! (.. right-sidebar -scrollTop) 0)))))
+
+
+;; TODO: temporary, and limits to one client opened.
+(def self-hosted-client (atom nil))
```

Comment:
I know this is temporary, but is this the same as `ws-connection`?

https://github.com/athensresearch/athens/blob/63aa3515ad71012f32ea0ff1776093e7c6cc1a62/src/cljs/athens/self_hosted/client.cljs#L18

Reply:
`component` and `re-frame` both frameworks. This is first instance of using `component` on front-end. How to use together if at all?